extensions/amp-image-slider/0.0/test/test-amp-image-slider.js

```diff
@@ -0,0 +1,210 @@
+/**
```

Comment:
Let's get this test suite to a place where we don't have uncommented code, and all the tests either pass or are skipped. It's fine to delete code, and we can make a point of adding tests in a later PR.